language/resource.language.fi_fi/strings.po

```diff
@@ -0,0 +1,2420 @@
+msgctxt "#31511"
+msgid "Shortcut"
+msgstr "Pikakuvake"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31512"
+msgid "Classic mode"
+msgstr "Perinteinen tyyli"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31513"
+msgid "Side Menu Icons"
+msgstr "Sivuvalikon kuvakkeet"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31514"
+msgid "Global Widgets"
+msgstr "Globaalit widgetit"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31515"
+msgid "Main Items"
+msgstr "Päänimikkeet"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31516"
+msgid "Edit items"
+msgstr "Muokkaa nimikkeitä"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31517"
+msgid "Reverse order"
+msgstr "Käänteinen järjestys"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31518"
+msgid "Highlight type"
+msgstr "Korostuksen tyyppi"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31521"
+msgid "Edit global widgets"
+msgstr "Määritä globaalit widgetit"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31533"
+msgid "Add list"
+msgstr "Lisää lista"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31534"
+msgid "Del list"
+msgstr "Poista lista"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31535"
+msgid "Restore"
+msgstr "Palauta"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31536"
+msgid "Choose fallback image folder"
+msgstr "Valitse varakuvien kansio"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31537"
+msgid "Edit submenu items"
+msgstr "Muokkaa alavalikon nimikkeitä"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31541"
+msgid "Next episode"
+msgstr "Seuraava jakso"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31542"
+msgid "Last episode"
+msgstr "Viimeinen jakso"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31543"
+msgid "Anticipated"
+msgstr "Odotettu"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31544"
+msgid "Airing"
+msgstr "Jatkuu"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31545"
+msgid "Premiering"
+msgstr "Ensi-illassa"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31546"
+msgid "TBC"
+msgstr "Kerrotaan myöhemmin"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31547"
+msgid "Yesterday"
+msgstr "Eilen"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts_Window.xml
+msgctxt "#31551"
+msgid "Use as home"
+msgstr "Käytä kotivalikkona"
+
+#: /1080i/Custom_1175_Dialog_Discover.xml
+msgctxt "#31553"
+msgid "Query"
+msgstr "Haku"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31554"
+msgid "Wall Poster"
+msgstr "Seinä (juliste)"
+
+#: /1080i/Includes_Views_Wall.xml
+msgctxt "#31555"
+msgid "Wall Landscape"
+msgstr "Seinä (vaakakuva)"
+
+#: /1080i/Includes_Views_Wall.xml
+msgctxt "#31556"
+msgid "Wall Square"
+msgstr "Seinä (neliö)"
+
+#: /1080i/Includes_Views_Wall.xml
+msgctxt "#31557"
+msgid "Wall Circle"
+msgstr "Seinä (ympyrä)"
+
+#: /1080i/Includes_Actions.xml
+msgctxt "#31558"
+msgid "Signpost"
+msgstr "Juliste ja teksti"
+
+#: /1080i/Includes_Views_List.xml
+msgctxt "#31559"
+msgid "List Details"
+msgstr "Lista Tiedot"
```

Comment:
I don't speak Finnish but should this be `Lista (tiedot)` instead? 

To match the way you've presented the other viewtype names like `Seinä (tiedot)` and `Rivi (tiedot)` etc.